src/shared/util/attributeSpanProcessor.ts

```diff
@@ -0,0 +1,16 @@
+import { BatchSpanProcessor, Span } from "@opentelemetry/sdk-trace-base";
+import { SpanExporter } from "@opentelemetry/sdk-trace-node";
+import { context } from "@opentelemetry/api";
+
+export class AttributeSpanProcessor extends BatchSpanProcessor {
```

Comment:
You should be implementing SpanProcessor from @opentelemetry/sdk-trace-base
https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/monitor/monitor-opentelemetry/src/traces/spanProcessor.ts